dev/benchmarks/macrobenchmarks/lib/src/web/recorder.dart

```diff
@@ -1253,7 +1257,7 @@ void startMeasureFrame(Profile profile) {
 
   if (!profile.isWarmingUp) {
     // Tell the browser to mark the beginning of the frame.
-    html.window.performance.mark('measured_frame_start#$_currentFrameNumber');
+    web.window.performance.mark('measured_frame_start#$_currentFrameNumber'.toJS);
```

Comment:
In dart2wasm, converting these keys to JS strings here seems like it would have a measurable effect on the actual performance here. Could we store all of these in globals as `JSString`s so that we only do the conversion a single time, and it's less likely to cause noise in our perf numbers?

Reply:
We spoke offline, the use of string interpolation here makes this non-trivial.

We're going to land this as is, and then revisit if it shows up on profiles.